soc/nuvoton/numicro/m48x/soc.c

```diff
@@ -12,6 +12,11 @@ void soc_reset_hook(void)
 {
 	SYS_UnlockReg();
 
+	/* DISABLE SPIM CACHE */
```

Comment:
Nit:

```suggestion
	/* Disable SPIM cacge */
```

Maybe even add a line explaining why